data/src/main/java/co/nimblehq/compose/crypto/data/ResponseMapping.kt

```diff
@@ -0,0 +1,41 @@
+package co.nimblehq.compose.crypto.data
+
+import co.nimblehq.compose.crypto.data.model.response.ErrorResponse
+import co.nimblehq.compose.crypto.data.service.JsonApiException
+import co.nimblehq.compose.crypto.data.service.UnknownException
+import co.nimblehq.compose.crypto.data.service.providers.MoshiBuilderProvider
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.transform
+import retrofit2.Response
+
+@OptIn(FlowPreview::class)
```

Comment:
Can we remove this 🙏 
```suggestion
@OptIn(FlowPreview::class)
```

Reply:
This annotation requires when we use `transform` 😄

Reply:
Hmm 🤔  I got this warning on my side that's why I think we no need to use it, Did you get it @hoangnguyen92dn 
![Screen Shot 2565-08-15 at 14 26 44](https://user-images.githubusercontent.com/28002315/184594093-abc6ed59-8392-4606-a431-a54a29e6b163.png)
?

Reply:
@Wadeewee Not sure why it requires me to add the annotation 😄 We can safety remove it for now 👍